### ISSUE TITLE
Npm start not working

### ISSUE THREAD

**Issue by cabb** [adminkit/adminkit#118]
I was not able to start the project with the configured packages and their versions.

## Current Behavior 😯

The project is not running  with `npm start`.
It receives the following error:
```
$ npm run start

> @adminkit/core@3.1.0 start
> webpack serve --config webpack.config.js --progress

[webpack-cli] Unable to load '@webpack-cli/serve' command
[webpack-cli] TypeError: options.forEach is not a function
    at WebpackCLI.makeCommand (/Users/workspace/frontend/adminkit/node_modules/webpack-cli/lib/webpack-cli.js:173:25)
    at ServeCommand.apply (/Users/workspace/frontend/adminkit/node_modules/@webpack-cli/serve/lib/index.js:41:19)
    at loadCommandByName (/Users/workspace/frontend/adminkit/node_modules/webpack-cli/lib/webpack-cli.js:909:35)
    at async Command.<anonymous> (/Users/workspace/frontend/adminkit/node_modules/webpack-cli/lib/webpack-cli.js:1485:17)
    at async Promise.all (index 0)
    at async WebpackCLI.run (/Users/workspace/frontend/adminkit/node_modules/webpack-cli/lib/webpack-cli.js:1523:9)
    at async runCLI (/Users/workspace/frontend/adminkit/node_modules/webpack-cli/lib/bootstrap.js:11:9)
```

## Expected Behavior 🤔

The project should correctly start with `npm start`

## Steps to Reproduce 🕹

Steps:

1. check out the project
2. perform `npm i`
3. perform `npm start`
4. check the command line output

## Your Environment 🌎

- OS: MacOS
- node: 16.13.0
- npm: 8.3.1

## Additional Information 

I took a quick check on the options variable in webpack-cli. And somehow the options variable is a promise (rejected) and not an array. Upgrading the dependency `webpack-cli` to `4.9.1` solves it for me.

**Comment by paullaros** [comment]
Resolved in #121

**Comment by cabb** [comment]
Works as expected
